astropy/coordinates/builtin_frames/galactic_transforms.py

```diff
@@ -23,7 +23,7 @@ def fk5_to_gal(fk5coord, galframe):
     mat2 = rotation_matrix(90 - Galactic._ngp_J2000.dec.degree, 'y')
     mat3 = rotation_matrix(Galactic._ngp_J2000.ra.degree, 'z')
 
-    return mat1 * mat2 * mat3 * pmat
+    return matmul(matmul(matmul(mat1, mat2), mat3), pmat)
```

Comment:
You could wrap it in an `if py35: eval("a @ b")` but granted that's not a good choice. But the `@` operator is really bad for backwards-compatibility because implementing it means that older python are not just not supported anymore but it will be impossible to use it because of the SyntaxErrors. So maybe using `eval()` until 2025 would be the best way if astropy doesn't want to **exclude** any older python version.